src/main/java/org/folio/bulkops/util/UpdateOptionTypeToFieldResolver.java

```diff
@@ -71,10 +71,8 @@ public static String getFieldByUpdateOptionType(UpdateOptionType type, EntityTyp
       return "Staff suppress";
     } else if (ITEM_NOTE == type) {
       return "Notes";
-    } else if (ADMINISTRATIVE_NOTE == type && ITEM == entity) {
-      return "Administrative Notes";
-    } else if (ADMINISTRATIVE_NOTE == type && HOLDINGS_RECORD == entity) {
-      return "Administrative notes";
+    } else if (ADMINISTRATIVE_NOTE == type && (ITEM == entity || HOLDINGS_RECORD == entity)) {
```

Comment:
any reason to check entity? you mad column name the same across them

Reply:
@NikitaSedyx 
the key reason to avoid code refactoring is decreasing regression testing efforts in the scope of the exact dev task.

Reply:
fixed